code/mob.dm

```diff
@@ -2394,7 +2394,8 @@
 // jitteriness - copy+paste of dizziness
 
 /mob/proc/make_jittery(var/amount)
-	if (!ishuman(src)) // for the moment, only humans get dizzy
+	// usually, only humans get jittery. watchful eyes also get jittery now.
+	if (!ishuman(src) && !istype(src, /mob/living/critter/small_animal/floateye/watchful))
```

Comment:
please fix this instead of adding to jank

Reply:
Just remove the check, if you want the eyes to become non-jittery again then make sure they have `/datum/lifeprocess/disability`

Reply:
removed the check altogetherthen . hope nothing breaks